stellarphot/gui_tools/comparison_functions.py

```diff
@@ -330,82 +333,46 @@ def _set_object(self):
         Try to automatically set object name immediately after file is chosen.
         """
         try:
-            self.object_name.value = self._file_chooser.header["object"]
+            self.object_name.value = (
+                f"<h2> Object: {self._file_chooser.header['object']}</h2>"
+            )
+            self._object = self._file_chooser.header["object"]
         except KeyError:
-            # No object, will show empty box for name
+            # No object, will show placeholder
+            self.object_name.value = "<h2>Object unknown</h2>"
             self.object_name.disabled = False
```

Comment:
Seems weird that `self.object_name.disabled = False` here, is that correct?  I assume this is because you used to have an empty box here... shouldn't you just remove the `self.object_name.disabled` variable or is it used elsewhere?

Reply:
Yeah, should be axed